apps/arweave/src/ar_randomx_state.erl

```diff
@@ -1,6 +1,6 @@
 -module(ar_randomx_state).
 
--export([init/2, init/4, start/0, start_block_polling/0, reset/0, hash/2,
+-export([init/2, init/4, start/0, start_block_polling/0, reset/0,
```

Comment:
I would remove this module completely because we do not need it any more. The `ar_randomx_state:get_key_block` query in `ar_http_iface_middleware` is totally unnecessary and easy to remove too.

Reply:
Sounds good! Can I do it in the next PR?

Reply:
Off course